src/test/kotlin/de/itemis/mps/gradle/test/CreateDmgTest.kt

```diff
@@ -0,0 +1,161 @@
+package de.itemis.mps.gradle.test
```

Comment:
I really like the idea having a minimal RCP test which can be executed. 
Could you maybe just drop some lines in the readme about the test idea and the setup? 
This would be great and could help others in the future to update/enrich the test if required.

Reply:
I'm puzzled here. Can you please clarify what is not clear?

I'm using a standard approach to use Gradle plugins: org.gradle.testkit.runner.GradleRunner
The test creates Gradle project within a temporary directory and executes a single task there.